src/nix/profile.cc

```diff
@@ -329,7 +335,63 @@ struct CmdProfileInstall : InstallablesCommand, MixDefaultProfile
             manifest.elements.push_back(std::move(element));
         }
 
-        updateProfile(manifest.build(store));
+        try {
+            updateProfile(manifest.build(store));
+        } catch (BuildProfileConflictError & conflictError) {
+            std::string fallbackInstallable = "INSTALLABLE";
+            std::string conflictingInstallable = [&]() {
+                auto maybeConflictingStorePath = StorePath::maybeParseAbsolutePath(conflictError.conflictingPkgPath);
```

Comment:
```suggestion
                auto maybeConflictingStorePath = store->maybeParseStorePath(conflictError.conflictingPkgPath);
```

Reply:
Oh, that is a lot more practical than introducing parseAbsolutePath :sweat_smile: Thanks :+1: